packages/main/src/components/protected/topbar/components/Profile.jsx

```diff
@@ -46,7 +46,7 @@ const Profile = () => {
   useEffect(() => {
     if (currentWorkspace) {
       authAxios
-        .get(`/organizations/${currentWorkspace}`)
+        .get(`/organization/${currentWorkspace}`)
```

Comment:
This should be `/organisations/${currentWorkspace}`

Reply:
Seen